index.js

```diff
@@ -168,9 +165,6 @@ class IgnitorPlugin {
 
   wrap() {
     const { functions } = this.options();
-
-    // make packageDir where wrappers are placed
-    utils.mkdir(this.packageDir);
```

Comment:
moved to the hooks section, `build.prebuild` is now called before the wrap function in the `ignitor:wrap:wrap` event.